advanced_functionality/pytorch_deploy_large_GPT_model/GPT-J-6B-model-parallel-inference-DJL.ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<p>Can't the provided execution role be the same as the current notebook execution role, so the user doesn't need to find some other ARN?</p><p> </p><p>Try to programmatically pass in the image ID and S3 path similar to the previous references.</p><p> </p><p>Is there a reason why you're using the AWS CLI instead of the SageMaker Python SDK? The SDK is typically used in notebooks and more widely used by customers.</p><p> </p><p>Please use the page title in link text - ie <code>SageMaker Roles documentation</code> rather than just <code>documentation</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3540/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>SDK for the new capability is not yet ready. that is why we use CLI. </p>